src/robusta/integrations/kubernetes/custom_models.py

```diff
@@ -226,6 +226,7 @@ def create_debugger_pod(
                 namespace=INSTALLATION_NAMESPACE,
             ),
             spec=PodSpec(
+                serviceAccountName="robusta-runner-service-account",
```

Comment:
Will this name be correct if someone changed the helm release name to something other than `robusta`?